pkg/translator/azure/go.mod

```diff
@@ -0,0 +1,53 @@
+//module github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/azure
+
+go 1.20
+
+require (
+	github.com/json-iterator/go v1.1.12
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatatest v0.85.0
+	github.com/relvacode/iso8601 v1.3.0
+	github.com/stretchr/testify v1.8.4
+	go.opentelemetry.io/collector/component v0.85.0
+	go.opentelemetry.io/collector/pdata v1.0.0-rcv0014
+	go.opentelemetry.io/collector/semconv v0.85.0
+	go.uber.org/zap v1.25.0
+	golang.org/x/exp v0.0.0-20230905200255-921286631fa9
```

Comment:
The failing test can be resolved by downgrading this dep to v0.0.0-20230713183714-613f0c0eb8a1 for now. The new version isn't required for this PR